data/ka'het/ka'het missions.txt

```diff
@@ -92,42 +92,107 @@ mission "First Contact: Ka'het: Remnant 1B"
-			`	Apart from the utter silence, which is pretty common in most rings, you are convinced by the fact that every room, apartment and cabin has been carefully emptied, leaving nothing behind. Perhaps it would be wise to ask the Quarg what happened here...`
-				accept
+			`Do you want to show the Remnant the piece of debris you found in the abandoned, empty system, and ask them what they know about it?`
+			choice
+				`	(Yes.)`
+				`	(Not now.)`
+					defer
+			`	A group of Remnant researchers quickly come to your ship when you mention you've found an odd piece of technology, but look disapointed and some even frown when they see the debris, as you explain the system you found it and how it reacted to being fed some energy.`
+			`	"We've found many fragments like this one drifting in that system over the decades. It could maybe improve our cooling systems, but we could never begin to comprehend how it works," one of them says.`
+			`	They say they don't know any more than you do about the object, but ask that you share with them any information on how it works, should you find out.`
+				decline
+
+mission "Ringworld Debris: Hai"
+	invisible
+	landing
+	source
+		government "Hai"
+	to offer
+		has "ringworld debris"
+		has "First Contact: Hai: offered"
+	on offer
+		conversation
+			`Do you want to show the Hai the piece of debris you found in the abandoned, empty system, and ask them what they know about it?`
+			choice
+				`	(Yes.)`
+				`	(Not now.)`
+					defer
+			`	You ask around for a bit, and eventually are lead to a workshop where some Hai engineers and shipwrights are gathered.`
+			`	"Greetings, human friend! How may we help you?"`
+			`	You show them the piece of debris, and they observe it for a bit, but after just a few minutes of handling it they conclude they don't have the necessary equipment to analyze it.`
+			`	"Sorry, whatever it is that you've found, it is too advanced a technology for our systems," they say. You thank them for their time, and head back to your ship.`
+				decline
 
-mission "First Contact: Z Ka'het: Quarg 2"
+mission "Ringworld Debris: Coalition"
+	invisible
 	landing
+	source
+		government "Coalition" "Heliarch"
+	to offer
+		has "ringworld debris"
+		has "Coalition: First Contact: done"
+	on offer
+		conversation
+			`Do you want to show the members of the Coalition the piece of debris you found in the abandoned, empty system, and ask them what they know about it?`
+			choice
+				`	(Yes.)`
+				`	(Not now.)`
+					defer
+			`	After asking around, you're directed to a large Heliarch building, where several of them are standing guard, weapons on hand, while others are performing maintenance on a chunk of hull plating. Judging by its size, it may be a warehouse for their weapons and outfits. "Of assistance how may we be, kind visitor?" One of the agents asks.`
```

Comment:
```suggestion
			`	After asking around, you're directed to a large Heliarch building, where several of them are standing guard, weapons in hand, while others are performing maintenance on a chunk of hull plating. Judging by its size, it may be a warehouse for their weapons and outfits. "Of assistance how may we be, kind visitor?" One of the agents asks.`
```